robots/cmd/flake-report-creator/README.md

```diff
@@ -0,0 +1,66 @@
+flake-report-creator
+====================
+
+aims to use the flakefinder logic of creating reports for a set of GCS job directories, whereas it just fetches all subdirectories, tries to find junit files and returns a report of the aggregated data. In general we can  generate a matrix of flakiness over several runs of several jobs.
+
+This tool enables a user to create an ad-hoc report of any GCS directories that contain kubevirt testing junit files, in turn enabling a report over unrelated or just selected PRs, or a selected set of periodic jobs. The user needs to make sure whether that makes sense of course 😏
+
+**Note: job results older than two weeks are skipped by default. Use `--start-from` argument to extend the range to older results**
+
+Usage
+-----
+
+**Example 1:** we want to create a report from all job runs of two PRs on openshift-ci.  But we want to skip all jobs that don't match a certain regular expression.
+
+*NB1: we use `start-from` here as some test results would have been skipped, as this was a quite long running PR*
+*NB2: we use `sub-dir-regex` here to filter out other unimportant jobs (openshift-ci related validation checks i.e)*
+
+```bash
+$ bazel run //robots/cmd/flake-report-creator -- \
```

Comment:
How abou simplifying these very technical calls with a simpler structure for users? Something like

```
--ci-system kubevirt --presubmits 123 456 789
--ci-system openshift --periodics release-0.44 release-0.43
```

Otherwise I think this quite useful idea is too hard to use.